### ISSUE TITLE
Incorrect npm name

### ISSUE THREAD

**Issue by raphacmartin** [rwaltenberg/angular-money-mask#13]
Hi, thanks for this directive!

When I ran `npm install angular-money-directive --save` as the README.md says to do, but it installed this plugin: https://github.com/fiestah/angular-money-directive

The correct command is` npm install angular-money-mask --save` I think, isn't it?

Thank you!

**Comment by rwaltenberg** [comment]
You're totally right! `angular-money-directive` is a former name. Could you please send a PR with that change to the `README.md` file?
